tests/index.test.ts

```diff
@@ -110,7 +110,7 @@ describe('Toolkit#constructor', () => {
   it('exits if the event is not allowed with an array of events', () => {
     // tslint:disable-next-line:no-unused-expression
     new Toolkit({ logger, event: ['pull_request'] })
-    expect(process.exit).toHaveBeenCalledWith(1)
+    expect(process.exit).toHaveBeenCalledWith(78)
```

Comment:
Can you change this (and the other tests) to use the exported `NeutralCode` from `src/exit.ts`? It's a more "type-safe" way of verifying that the expected value was used.

So this'd need `import { NeutralCode } from '../src/exit'` at the top of the file, and:

```suggestion
    expect(process.exit).toHaveBeenCalledWith(NeutralCode)
```

Reply:
That's definitely better! (and thanks suggestion too) I've changed in the local and squashed.